src/txn/txn_rollback_to_stable.c

```diff
@@ -254,10 +254,15 @@ __rollback_row_ondisk_fixup_key(WT_SESSION_IMPL *session, WT_PAGE *page, WT_ROW
 
         /*
          * Do not include history store updates greater than on-disk data store version to construct
-         * a full update to restore. Comparing with timestamps here has no problem unlike in search
-         * flow where the timestamps may be reset during reconciliation. RTS detects an on-disk
-         * update is unstable based on the written proper timestamp, so comparing against it with
-         * history store shouldn't have any problem.
+         * a full update to restore. Even if we include the most recent updates than the on-disk
```

Comment:
Even if we include the most recent updates than the on-disk version shouldn't be problem -> Including more recent updates than the on-disk version shouldn't be a problem

Reply:
Fixed.